hermes/probe/read/read_file.go

```diff
@@ -0,0 +1,80 @@
+// TODO (#70) add license header and author line
+// TODO (#69) add doc strings
+
+package read
+
+import (
+	"context"
+	"crypto/sha1"
+	"fmt"
+	"io"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/google/cloudprober/logger"
+	"github.com/googleapis/google-cloud-go-testing/storage/stiface"
+	"github.com/googleinterns/step224-2020/hermes/probe"
+	m "github.com/googleinterns/step224-2020/hermes/probe/metrics"
+)
+
+func checkSumFromName(fileName string) (string, error) {
```

Comment:
You only use this in one place, and it's pretty simple, so maybe just inline it?

Reply:
inlined it